lib/overcommit/hook/shared/rake_target.rb

```diff
@@ -0,0 +1,24 @@
+module Overcommit::Hook::Shared
+  # runs specified rake targets. It fails on the first non-
+  # successfull exit.
+  #
+  module RakeTarget
+    def run(targets = config['targets'])
```

Comment:
It's a bit odd to define a parameter list here when the `Overcommit::Hook::Base#run` method does not define any parameters. You might as well pull this down into the method implementation.